### ISSUE TITLE
Can this project be released and get a licence?

### ISSUE THREAD

**Issue by hustfly** [chenfengjw163/vue-json-viewer#8]
So others can use it as dependency, work with npm.

**Comment by chenfengjw163** [comment]
OK, MIT license has been added

**Comment by hustfly** [comment]
There is no release, how can I add into webpack?

**Comment by chenfengjw163** [comment]
Has been released to npm version number is 2.0.5, npm package contains dist file, you only need to require or import vue-json-viewer in the project：
``` javascript
import JsonViewer from 'vue-json-viewer' 
```
or
``` javascript
const JsonViewer = require('vue-json-viewer')
```
